internal/venice-test-common/src/integrationTest/java/com/linkedin/venice/controller/TestEnablingSSLPushInVeniceHelixAdminWithIsolatedEnvironment.java

```diff
@@ -23,7 +24,7 @@ public class TestEnablingSSLPushInVeniceHelixAdminWithIsolatedEnvironment extend
 
   @BeforeMethod(alwaysRun = true)
   public void setUp() throws Exception {
-    setupCluster(false);
```

Comment:
Is this needed? I don't see the previous version of the constructor deprecated. Why no leave it as is?

Reply:
It is removed in `AbstractTestVeniceHelixAdmin`. I was touching that file, and felt it was not worth having 3 constructors as they were quite basic, and were test-only. I can revert it, though if you feel otherwise. I don't have strong opinions for this

Reply:
in that case, can you also remove the constructor that is not needed.